core/scripts/vrfv2/testnet/main.go

```diff
@@ -658,6 +673,9 @@ func main() {
 		helpers.ParseArgs(createSubCmd, os.Args[2:], "coordinator-address")
 		coordinator, err := vrf_coordinator_v2.NewVRFCoordinatorV2(common.HexToAddress(*coordinatorAddress), ec)
 		helpers.PanicErr(err)
+		payload, err := coordinatorV2ABI.Pack("createSubscription")
```

Comment:
This extra code will need to be added to every func we want to use on harmony right?

What if we just add an extra GAS_LIMIT environment variable that, if set, is used? In reality for these funcs it's fine just to set a high gas limit, there's no chance they use a bunch of extra gas anyway

Reply:
I like that idea. setting env variable seems a lot simpler to maintain especially since this special handling is needed only for harmony right now.

Reply:
While that is doable I feel like it reduces the utility of this script on Harmony if you have to specify a GAS_LIMIT manually for each call. 

Maybe we should estimate gas manually as we are doing now in this PR for all of the non-variable gas operations (e.g creating a subscription, funding, etc.) and store those estimates (plus some buffer) in a map and look them up before executing the tx?

```golang
var gasLimits = map[string]uint64{
  "addConsumer": ...,
  "createSubscription": ...,
  // ... and so on ...
}

owner.GasLimit = gasLimits["addConsumer"]
```

This is still not great IMO. There has to be a way with less friction.

Reply:
Are there benefits to estimating gas limit precisely for each operation? I'd prefer setting high GAS_LIMIT env variable just once (or set a high default value), then for each operation, the excess gas will be returned after TX is created.

Reply:
Yes, the benefits are that it's more likely to get included as a transaction than if you set an exceedingly high gas limit, and also that you don't have to try to guess from little to no information. Estimate gas gives a very precise estimate that is almost always right, however we sometimes bump it for VRF in the event of request fulfillment.

That said, I think if we were to add a GAS_LIMIT environment variable, it should override whatever EstimateGas does, not completely replace it. That way you can just put it in one place before any command is executed (above the switch statement) and the rest of the code can remain as-is.